docs/configuration.rst

```diff
@@ -166,6 +166,59 @@ Data governance configuration
 **dp** can sends **dbt** metadata to DataHub. All related configuration is stored in ``config/<ENV>/datahub.yml`` file.
 More information about it can be found `here <https://datahubproject.io/docs/metadata-ingestion#recipes>`_ and `here <https://datahubproject.io/docs/generated/ingestion/sources/dbt>`_.
 
+Data ingestion configuration
+++++++++++++++++++++++++++++++
+
+Ingestion configuration is divided into two levels:
+
+- General: ``config/<ENV>/ingestion.yml``
+- Ingestion tool related: e.g. ``config/<ENV>/airbyte.yml``
+
+``config/<ENV>/bi.yml`` contains basic configuration of ingestion:
+
+.. list-table::
+   :widths: 25 20 55
+   :header-rows: 1
+
+   * - Parameter
+     - Data type
+     - Description
+   * - enable
+     - bool
+     - Flag for enable/disable ingestion option in **dp**.
+   * - engine
+     - enum string
+     - BI tool you want to working with (currently only supported value is ``airbyte``).
+
+``config/<ENV>/airbyte.yml`` must be present if chosen engine is ``airbyte``. It is used to upsert connections
+in Airbyte and then by Airflow ingestion Airflow task executor preceding dp build.
+
+.. list-table::
+   :widths: 25 20 55
+   :header-rows: 1
+
+   * - Parameter
+     - Data type
+     - Description
+   * - airbyte_connection_id
+     - string
+     - name of Airbyte connection in Airflow
+   * - airbyte_url
+     - string
+     - https address of Airbyte deployment that allows to connect to Airbyte API
+   * - connections
+     - array<*connection*>
+     - configurations of Airbyte connection that should be upserted during CI/CD. Documented below
+   * - tasks
+     - array<*task*>
+     - configurations of Airflow tasks used by `dbt-airflow-factory <https://github.com/getindata/dbt-airflow-factory>`_.
+       Allowed *task* options are documented `here <https://dbt-airflow-factory.readthedocs.io/en/latest/configuration.html#id3>`_.
```

Comment:
is this configuration up-to-date? https://dbt-airflow-factory.readthedocs.io/en/latest/configuration.html#id3

Reply:
afaik it is. Even with a typo `asyncronous` that is now widespread